tests/other/metadce/minimal_main_Oz_USE_PTHREADS_PROXY_TO_PTHREAD.size

```diff
@@ -1 +1 @@
-16368
```

Comment:
Can this file be reverted now without causes a test failure?  (looks like its within the tolerance maybe?).

Reply:
It seems to be within the expected tolerance. I reverted it to its original state with commit 4f5be19df19e2967642e93d3aaf1ec8369d005d4.